client/src/components/Authorization.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import axios from 'axios';
+
+
+const Authorization = {
+  isAuthenticated: false,
+  authenticate(cb) {
+    fetch('http://localhost:55555/user/login', {
+      method: 'POST',
+      body:JSON.stringify({'username':localStorage.getItem('username') , 'password':localStorage.getItem('password')})
```

Comment:
why you don't take `username` and `password` as parameters instead to get them from localStorage here and you will get a better abstraction with this component?

Reply:
se pueden pasar comentarios a authenticate()??